client/src/_actions/auth.js

```diff
@@ -0,0 +1,35 @@
+import dayjs from 'dayjs'
+import jwt_decode from 'jwt-decode'
+
+export const ADD_AUTH = 'ADD_AUTH'
+export const REMOVE_AUTH = 'REMOVE_AUTH'
+
+export function addAuth(token) {
+  const expiration = dayjs.unix(jwt_decode(token).exp).toDate()
```

Comment:
generally, these action methods are to be really, really simple and all the data/state changes should happen in the reducer. therefore, the logic for creating the expiration time, setting localStorage ,and setting the token to `null` done below should be relocated to the reducer.